### PR TITLE
Console: Use proper transaction handling

### LINKED ISSUE #2109: hyriseConsole: Use new transaction handling
We should use the transaction handling introduced by #1993 instead of having explicit `begin` etc. commands.

As part of this, remove `SQLPipeline::failed_pipeline_statement` or make it private. We shouldn't expose single SQLPipelineStatements (see #2111).

### PR DESCRIPTION
**Pull request by mrks**
Instead of having its own hacky hacky functions, the console now uses the transaction handling as introduced in #1993.

fixes #2109

### CODE REVIEW

**Comment by mrks** [comment]
This was even easier than expected :)

File: scripts/test/hyriseConsole_test.py [hunk 1]
```diff
@@ -62,6 +62,22 @@ def main():
...
+	console.sendline("select sum(a) from test")
+	console.expect("860")
+	console.sendline("txinfo")
+	console.expect("Active transaction")
+	console.sendline("insert into test (a) values (21); rollback;")
+	console.sendline("select sum(a) from test")
+	console.expect("840")
+	console.sendline("txinfo")
+	console.expect("auto-commit mode")
+
```
Comment (klauck) [inline scripts/test/hyriseConsole_test.py:80]: Should we also test invalid schedules of transaction statements, e.g., `begin;begin;`?

**Comment by mrks** [comment]
I will simply remove all the code that deals with transaction conflicts. That cannot occur in the console.

**Comment by mrks** [comment]
@Bensk1 - just remembered that @klauck isn't working right now. Could you take over, please? He already looked at the transaction part, so there shouldn't be too much left over.

**Pull request merged**